zproject/urls.py

```diff
@@ -310,6 +311,10 @@
         DELETE=deactivate_user_backend,
     ),
     rest_path("users/<int:user_id>/subscriptions/<int:stream_id>", GET=get_subscription_backend),
+    # Miscellaneous endpoint for realm owners to update emails of other users.
+    rest_path(
+        "users/change_email", PATCH=(change_user_email_backend, {"intentionally_undocumented"})
```

Comment:
I think probably we should do API documentation for this? 

I'm also not sure if the URL should be `/users/{email}/change_email`; that would be more REST-like, at least.

Though maybe even cleaner would be to just have a `PATCH /users/{email}` that calls `update_user_backend`, and that function should allow changing the email if the acting user has the relevant capability?

Reply:
Yeah, added the new endpoint as discussed in the czo thread, with docs now.